expan/core/early_stopping.py

```diff
@@ -228,26 +250,37 @@ def bayes_factor(x, y, distribution='normal', num_iters=25000):
         num_iters: number of iterations of bayes sampling
 
     Returns:
-        tuple: 
-            - stop label
-            - effect size (delta)
-            - credible interval of delta
-            - sample size of x
-            - sample size of y
-            - absolute mean of x
-            - absolute mean of y
+        dictionary with statistics
     """
     traces, n_x, n_y, mu_x, mu_y = _bayes_sampling(x, y, distribution=distribution, num_iters=num_iters)
     kde = gaussian_kde(traces['delta'])
 
     prior = cauchy.pdf(0, loc=0, scale=1)
     # BF_01
     bf = kde.evaluate(0)[0] / prior
-    stop = int(bf > 3 or bf < 1 / 3.)
-    interval = HDI_from_MCMC(traces['delta'])
-
-    return stop, mu_x - mu_y, {'lower': interval[0], 'upper': interval[1]}, n_x, n_y, mu_x, mu_y
-
+    # stop = int(bf > 3 or bf < 1 / 3.)
+    stop = bf > 3 or bf < 1 / 3.
+
+    credibleMass = 0.95                # another magic number
+    leftOut      = 1.0 - credibleMass
+    p1           = round(leftOut/2.0, 5)
+    p2           = round(1.0 - leftOut/2.0, 5)
+    interval = HDI_from_MCMC(traces['delta'], credibleMass)
+
+    # return stop, mu_x - mu_y, {'lower': interval[0], 'upper': interval[1]}, n_x, n_y, mu_x, mu_y
+    return {'stop'     : stop,
+            'delta'    : mu_x - mu_y,
+            'interval' : {p1*100: interval[0], p2*100: interval[1]},
```

Comment:
we could do that, but floats look more compact to me and I don't see that they don't suffice